.github/workflows/example.yml

```diff
@@ -0,0 +1,28 @@
+name: Example SSL Test
+
+on:
+  workflow_dispatch:
+    inputs:
+      test_uri:
+        description: 'URI to test (e.g., https://example.com)'
+        required: true
+        default: 'https://example.com'
```

Comment:
Use https://expired.badssl.com/ instead of example.com, here and in descriptions and readme